Library/Source/NativeWindow.h

```diff
@@ -15,11 +15,26 @@ namespace Babylon
         static Napi::FunctionReference GetAToBFunction(Napi::ObjectReference& nativeWindow);
 
         NativeWindow(const Napi::CallbackInfo& info);
+        ~NativeWindow();
 
         void Resize(size_t newWidth, size_t newHeight);
 
         using OnResizeCallback = std::function<void(size_t, size_t)>;
-        using OnResizeCallbackTicket = arcana::ticketed_collection<OnResizeCallback>::ticket;
```

Comment:
The third and middle-est change removes the dependency on Arcana's `ticketed_collection` and just makes a small bespoke alternative instead. This was to get around the order-of-destruction problem described in #142, which we should still consider how to deal with long-term but may not be as tricky as I thought at first. In this particular case, all that's necessary to allow the `NativeWindow` and `NativeEngine` to be destroyed in arbitrary orders is for the `OnResizeCallbackTicket` to simply do nothing if the collection it came from was destroyed before it was. `arcana::ticketed_collection` doesn't support this, and adding that is a work-in-progress due to the fact that `ticketed_collection` is supposed to be thread-safe. But thread-safety isn't a concern here as these are both Babylon Native plugins that should only talk to each other on the same thread, so we can get away with doing something simpler. (And we can always move back to the Arcana type when it's ready, if we want to.)